guide/src/componentDocs/ButtonMenu/ButtonMenuDoc.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { P } from 'cyverse-ui';
+import meta from 'cyverse-ui/ButtonMenu.json';
+import { Figure, ComponentDoc, CodeBlock } from '../../components';
```

Comment:
Since `guide/src` is the defined [`context` in webpack](https://github.com/cyverse/cyverse-ui/blob/master/guide/webpack.config.js#L12) for the guide, I think you could do `... from 'components'` instead of directory traversal _up_. Not 💯% on that though ...